src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -451,6 +453,14 @@ const AppWrapper = (props: { children: ReactNode }) => {
     if (keriaConnectUrlRecord) {
       try {
         await Agent.agent.start(keriaConnectUrlRecord.content.url as string);
+
+        const pendingDeletions =
+          await Agent.agent.connections.getConnectionsPendingDeletion();
+
+        for (const id of pendingDeletions) {
+          await Agent.agent.connections.deleteConnectionById(id);
+          dispatch(removeConnectionCache(id));
```

Comment:
Also, we need to remove it from the UI immediately, and not show it in the UI if it's pending deletion. The only background task is removing it from KERIA and the DB

Reply:
This still isnt covered